.ci/travis/travis.sh

```diff
@@ -534,6 +534,43 @@ git-status)
   fi
   ;;
 
+check-since-version)
+  ## Travis merges the PR commit into origin/master
+  ## This identifies the PR's original commit
+  ## if it notices a merge commit
+  HEAD=`git rev-parse HEAD`
+  if git show --summary HEAD | grep ^Merge: ; then
+      echo "Merge detected."
+      HEAD=`git log -n 1 --no-merges --pretty=format:"%H"`
+  fi
+  ## Identify previous commit to know how much to examine
+  ## Script assumes we are only working with 1 commit if we are in master
+  ## Otherwise, it looks for the common ancestor with master
+  COMMIT=`git rev-parse $HEAD`
+  echo "PR commit: $COMMIT"
+
+  HEAD_NEW_FILES=$(git show $COMMIT | cat | grep -A 1 "\-\-\- /dev/null" | cat)
```

Comment:
Why `git show | cat`?
Is it the same as `git show --color=never`?
It seems to me that the second form is more clear.
The `grep` command also has this option.

Reply:
It is also `--no-pager`.